### PR TITLE
Add `mlstacks version` CLI command

### PR DESCRIPTION
**Pull request by strickvl**
- Added a `mlstacks version` command (+ tests)
- move unit tests into a `unit` folder.

(you can ignore all the moved test code. Just `tests/unit/cli` and the `src/` changes are what count in this PR.

## Pre-requisites

Please ensure you have done the following:

- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to docs, I have updated the documentation
      accordingly.
- [x] I have added tests to cover my changes.
- [x] I have based my new branch on `develop` and the open PR is targeting
      `develop`. If your branch wasn't based on develop read
      [Contribution guide on rebasing branch to develop](https://github.com/zenml-io/mlstacks/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      change)
- [ ] Other (add details above)

### CODE REVIEW

File: src/mlstacks/cli/cli.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 from typing import Optional
 
 import click
+import pkg_resources
```
Comment (fa9r) [inline src/mlstacks/cli/cli.py:21]: Does this require changes in the requirements of the package or does it already use setuptools elsewhere?

**Review by fa9r — APPROVED** [review]
LGTM

File: src/mlstacks/cli/cli.py (hunk 1 above)
Comment (strickvl) [reply src/mlstacks/cli/cli.py:21]: I don't think so. I did a fresh install of `mlstacks` in a new environment and it already had `setuptools` installed.

**Review by safoinme — APPROVED** [review]
LGTM!

**Pull request merged**